app/src/pages/inside/apiPage/apiPage.jsx

```diff
@@ -45,14 +45,18 @@ const OPTION_BLOCK_TAG_OPEN = '.opblock.is-open';
 const OPTION_BLOCK_SUMMARY = '.opblock-summary';
 
 @connect((state) => ({
-  token: tokenSelector(state),
+  tokenString: tokenSelector(state),
```

Comment:
If we already select the `tokenType` and `tokenValue` individually, does it make sense to not select them as a joined string additionally?
I suppose we can just join them right in the component if needed and omit `tokenSelector` usage here.